test/tests/functional/pbs_job_comment_on_resume.py

```diff
@@ -58,11 +58,10 @@ def test_job_comment_on_resume(self):
         jid1 = self.server.submit(J)
         self.server.expect(JOB, {'job_state': 'R'}, id=jid1)
         J = Job(TEST_USER, {'queue': 'expressq'})
-        J.set_sleep_time(1)
+        J.set_sleep_time(20)
         jid2 = self.server.submit(J)
         self.server.expect(JOB, {'job_state': 'S'}, id=jid1)
         self.server.log_match(
-            jid2 + ";dequeuing from expressq",
-            max_attempts=10)
+            jid2 + ";dequeuing from expressq")
```

Comment:
Now the job runs for 20sec, which means that by the time this log match is happening
this log match will be searching for a match for 20seconds before the job has even ended.  Perhaps you can delay the start of the log matching?  That way you'll get a full 60 seconds of being able to find a match.